tests/x11/vino.pm

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018 SUSE LLC
```

Comment:
How about you rename the test module to "vino_screensharing_available" to make it more obvious that you do not really test the VNC part but only the integration and configuration?

Reply:
Yes, I'll rename it and put it `x11/remote_desktop/vino_screensharing_available.pm` as you suggest. I'll also implement the other suggestions.